Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -21,6 +22,8 @@ open class Player: UIViewController {
                 vc.view.autoresizingMask = [.flexibleWidth, .flexibleHeight]
                 view.addSubview(vc.view)
                 vc.didMove(toParent: self)
+                
+                addTvRemoveGestureIfNeeded(vc)
```

Comment:
I think this method has more than one responsibility, maybe we can extract the check to another method.